public/app/features/datasources/components/DataSourcesList.tsx

```diff
@@ -83,6 +85,21 @@ export function DataSourcesListView({ dataSources, dataSourcesCount, isLoading,
                     dataSource.isDefault && <Tag key="default-tag" name={'default'} colorIndex={1} />,
                   ]}
                 </Card.Meta>
+                <Card.Tags>
+                  <LinkButton icon="apps" fill="outline" variant="secondary" href="/dashboard/new">
+                    Build a Dashboard
+                  </LinkButton>
+                  <LinkButton
+                    icon="compass"
+                    fill="outline"
+                    variant="secondary"
+                    className={styles.button}
+                    href={constructDataSourceExploreUrl(dataSource)}
+                    disabled={!canExploreDataSources}
```

Comment:
I am thinking about if it even makes sense to display the button if the user doesn't have the rights to use Explore 🤔 (My worry is that if the user only sees that it is "disabled" but they don't see the reason then it can get confusing)

Reply:
Done